src/Media/ImageUploaderService.php

```diff
@@ -74,6 +64,7 @@ public function __construct(
      * @inheritdoc
      */
     public function upload(
+        UUID $fileId,
```

Comment:
De uploader zelf is best verantwoordelijk voor het aanmaken van de UUID. Zo blijven we ook consistent met andere synchrone acties zoals het aanmaken van een evenement, zie: https://github.com/cultuurnet/udb3-php/blob/master/src/Event/EventEditingServiceInterface.php#L111-L120.